databricks/sdk/core.py

```diff
@@ -506,6 +508,56 @@ def __repr__(self) -> str:
         return f"<ConfigAttribute '{self.name}' {self.transform.__name__}>"
 
 
+class Cloud(Enum):
+    AWS = 'aws'
+    AZURE = 'azure'
+    GCP = 'gcp'
+
+
+@dataclass
+class DatabricksEnvironment:
+    cloud: Cloud
+
+    # domain suffixes are not very secret: https://crt.sh/?q=databricks
+    tld: str
+
+    azure_environment: AzureEnvironment = None
+
+    # The application (client) ID isn't a secret. See
+    # https://learn.microsoft.com/en-us/entra/identity-platform/developer-glossary#application-client-id
+    azure_application_id: str = None
+
+    def deployment(self, deployment_name: str) -> str:
+        return f'https://{deployment_name}{self.tld}'
```

Comment:
This is maybe just a bit abstract. It won't work in Azure the way a user might expect, where the DNS name can't be determined from the workspace ID alone. Given we're only using this for account host, I suggest we just inline it there.

Reply:
well... in this case we have to specify `adb-633....10` as deployment name

https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/account/workspaces.py#L101-L126

Reply:
Oh perfect, I actually didn't realize those workspace APIs worked in azure yet.